### DISCUSSION TITLE
UniGetUI 3.1.1 beta 3

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2534] [Announcements]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.1-beta3/total?style=for-the-badge&color=red)

## Changes 
 - Added PowerShell 7 Support.
 - Important changes in the UI backend:
   - Icons are loaded from a font file, instead of themed .png files.
   - Important improvements to package tags.
   - Improvements on Settings Page Controls
   - Package managers now have an optional DisplayName property
   - WinGet and Scoop icons have been updated to be coherent with their official icons
 - Package Bundles Page has been updated to inherit from the new AbstractPackagesPage
 - Improvements to how packages are imported and expoeted

SHA256: `C7A6A9BB90B9E1EFC3FC0FF09AA3DF217EF18288727E5F50D14A774F85F32EB5`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/UniGetUI/releases/tag/3.1.1-beta3'>UniGetUI 3.1.1 beta 3</a>.</em>

**Comment by Mertsch** [comment]
This is awesome!
Use "included winget" also works now. Only need it one system where the installed winget is not recognized for some reason (win server 2019)

**Reply by marticliment** [reply]
Windows Server support for WinGet is kinda broken, so expect it to not work

**Comment by BrianG61UK** [comment]
When there is a PIP update the package updates Widget stops working, just saying "An error occurred with this widget: CANNOT_FETCH_UPDATES: update fetching failed with code InternalServerError". I had to go to WingetUI itself to install the update, rather than just clicking on the button in the widget. Once the update is done, the widget starts working again!

**Reply by marticliment** [reply]
Fixed on code!

**Comment by nannerpusser** [comment]
Powershell 7 support is one of those things I didn't wanna ask about because I figured you have a whole workflow devoted to those reports, but kept hoping it'd make it in soon. And here we are.
